subsys/bluetooth/host/l2cap.c

```diff
@@ -2189,6 +2189,11 @@ static void l2cap_chan_le_recv(struct bt_l2cap_le_chan *chan,
 		return;
 	}
 
+	if (buf->len < 2) {
+		BT_WARN("Too short data packet");
+		return;
```

Comment:
Should the channel be disconnected, like it is for other error cases? sdu_len exceeds chan RX mtu for example?

Reply:
@joerchan agreed, updated now